app/js/components/CreateEscrow.jsx

```diff
@@ -0,0 +1,82 @@
+import React, {Component} from 'react';
+import {Alert, Button, Card, CardBody, CardHeader, CardTitle, Form, FormGroup, Input, Label} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+function pad(number) {
+  if (number < 10) {
+    return '0' + number;
+  }
+  return number;
+}
+
+function toInputDate(date) {
+  return date.getUTCFullYear() +
+    '-' + pad(date.getUTCMonth() + 1) +
+    '-' + pad(date.getUTCDate()) +
+    'T' + pad(date.getUTCHours()) +
+    ':' + pad(date.getUTCMinutes());
+}
+
+class CreateEscrowForm extends Component {
```

Comment:
What do you think of keeping a 1-1 relation between file name and exported component name?

Reply:
Oops. I changed the Component name and forgot to change the file name. Hopefully it doesn't mess up my other PRs